amd/src/custom.js

```diff
@@ -240,6 +243,11 @@ define(['jquery'], function($) {
                     reloadPreview();
                 });
 
+                $('#add_notification_sendnotifications').on('change', function() {
+                    // Checking specifically whether ticked/checked or not to ensure it's displayed correctly (not toggling).
+                    reloadPreview();
+                });
+
```

Comment:
Do we need to reload the preview?

Reply:
I'm not really sure, I mostly copied what was done for the "dismissible" and "icon" fields.
I'll test this next week.

Best,
Daniel

Reply:
this was removed in the last commit as it looks like it's not really neede, thank you!